content/SCALETutorials/Storage/Disks/ReplacingDisks.md

```diff
@@ -26,6 +26,21 @@ TrueNAS wipes the data on the replacement disk as part of the process.
 Disk replacement automatically triggers a pool resilver.
 {{< /hint >}}
 
+This tutorial includes instructions for replacing a failed disk in TrueNAS systems with and without an available hot spare.
```

Comment:
What if the users have systems which do not allow hot plug or hot swapping their JBOD HDDs? There should be a section describing the "offlining", detaching and rebooting process in this case, as not all will do this with their systems operational.